### PR TITLE
Add support for setting of different colors for container's border and title

### LINKED ISSUE #73: Support setting of different colors for container's border and title

### PR DESCRIPTION
**Pull request by chipmunkie**
Introduce options for setting border title color and border title focus color. 

Fixes #73

### CODE REVIEW

File: container/container_test.go [hunk 1]
```diff
@@ -1003,6 +1003,154 @@ func TestNew(t *testing.T) {
...
+				return ft
+			},
+		},
+		{
+			desc:     "inherits title color on left child container",
+			termSize: image.Point{10, 10},
+			container: func(ft *faketerm.Terminal) (*Container, error) {
+				return New(
+					ft,
+					Border(linestyle.Light),
```
Comment (mum4k) [inline container/container_test.go:1112]: Did we intend to set the `TitleColor()` option here to verify it gets inherited to the child container? As it stands, this test might be verifying that the child container inherits the default `BorderColor()` option.

**Review by mum4k — COMMENTED** [review]
Thank you for the contribution! Looks good, just one question about a test case.

**Comment by mum4k** [comment]
(optional) Consider adding a line about this new functionality into the [CHANGELOG](https://github.com/mum4k/termdash/blob/devel/CHANGELOG.md). We can do that in the same PR.

**Comment by chipmunkie** [comment]
Addressed the comment and updated the CHANGELOG.

**Review by mum4k — APPROVED** [review]

**Pull request merged**